runtime/doc/lsp.txt

```diff
@@ -749,6 +754,12 @@ document_symbol()                              *vim.lsp.buf.document_symbol()*
 formatting({options})                               *vim.lsp.buf.formatting()*
                 TODO: Documentation
 
+formatting_sync({options}, {timeout_ms})       *vim.lsp.buf.formatting_sync()*
+                Same as |vim.lsp.buf.formatting()| but synchronous.  Useful
+                for running on save, to make sure buffer is formatted prior
+                to being saved.  {timeout_ms} is passed on to
```

Comment:
NIT: Extra whitespace after period.

Reply:
Yeah I don’t normally distinguish sentence-final periods by putting two spaces after them, but I noticed this was the case at various places in the help file, so I thought maybe Neovim docs use this convention :) Still, I’m happy to get rid of them (there’s another instance in the previous chunk, I think). It’ll be somewhat inconsistent, though.

Reply:
Meh, it's fine. We're fixing some stuff with doc generation so this will go away later anyway. Ignore for now.